pkg/provider/resource_cluster.go

```diff
@@ -88,77 +88,77 @@ func (c *ClusterResource) Schema() *schema.Resource {
 				Required:    true,
 			},
 			"created_at": {
-				Description: "Cluster Creation Time",
+				Description: "Cluster Creation Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"deleted_at": {
-				Description: "Cluster Deletion Time",
+				Description: "Cluster Deletion Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"expired_at": {
-				Description: "Cluster Expiry Time",
+				Description: "Cluster Expiry Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"first_recoverability_point_at": {
-				Description: "Earliest Backup recover time",
+				Description: "Earliest Backup Recover Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"instance_type": {
-				Description: "Instance Type",
+				Description: "Instance Type. e.g. \"azure:Standard_D2s_v3\", \"aws:c5.large\".",
```

Comment:
```suggestion
				Description: "Instance Type. For example, \"azure:Standard_D2s_v3\" or \"aws:c5.large\".",
```

Reply:
e.g. -> For example, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/4de4c956dd0831008781a5f59b9011542e378c5a
Use of `or` instead of `,`, fixed at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/ad10738d8c184a41484778f34da4a27070eaa0c3